src/App.js

```diff
@@ -1,9 +1,13 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import { Routes, Route } from 'react-router-dom';
 import BookList from './components/BooksList';
 import Categories from './components/Categories';
+import { retrieveBooks } from './redux/books/books';
 
 function App() {
```

Comment:
- [ **OPTIONAL** ] What do you think if you use an **arrow function** instead of a normal function for ES6 best practices and good code refactoring